src/beautifiers/goimports.coffee

```diff
@@ -0,0 +1,20 @@
+###
+Requires https://godoc.org/golang.org/x/tools/cmd/goimports
+###
+
+"use strict"
+Beautifier = require('./beautifier')
+
+module.exports = class Goimports extends Beautifier
+  name: "goimports"
+  link: "https://godoc.org/golang.org/x/tools/cmd/goimports"
+  isPreInstalled: false
```

Comment:
If `--version` is not available then I've used `--help` and returned a fake version of `0.0.0` like https://github.com/Glavin001/atom-beautify/blob/master/src/beautifiers/beautysh.coffee#L14-L16